src/components/MapInput/index.js

```diff
@@ -102,17 +90,15 @@ const MapInput = ({ className, hasGPSControl, value, onChange, mapOptions, event
   );
 
   const onSelect = useCallback(
-    async option => {
+    option => {
       dispatch(
         setValuesAction({ location: option.data.location, address: option.data.address, addressText: option.value })
       );
 
-      const stadsdeel = configuration?.map?.options?.stadsdeel || await getStadsdeel(option.data.location);
-
       onChange({
         geometrie: locationTofeature(option.data.location),
         address: option.data.address,
-        stadsdeel,
+        stadsdeel: configuration.map?.municipality,
```

Comment:
It shouldn't be required any more to provide the API with the `stadsdeel` prop. Can you confirm?

Reply:
As a result `municipality` can be removed from `configuration.map`

Reply:
Thank you this what I was wondering about, will fix